### PR TITLE
fix the folder_name of box addon in log

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
fix the problem https://trello.com/c/1qyfOSRH/81-staging-box-add-on-linked-folder-logged-incorrectly.

<b>Changes</b>
Now the box folder name in log updated correctly.

### CODE REVIEW

**Comment by chennan47** [comment]
@sloria rdy for review

File: website/addons/box/utils.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def log(self, action, extra=None, save=False):
             'project': self.node.parent_id,
             'node': self.node._primary_key,
             'folder_id': self.node.get_addon('box', deleted=True).folder_id,
-            'folder': self.node.get_addon('box', deleted=True).folder_name
+            'folder': self.node.get_addon('box', deleted=True).folder_path
```
Comment (sloria) [inline website/addons/box/utils.py]: It wouldn't hurt to leave `folder` and add `folder_path` as an additional item in order to maintain backwards compatibility.

Comment (chennan47) [inline website/addons/box/utils.py]: addressed in  146f710

File: website/addons/box/utils.py [hunk 2]
```diff
@@ -46,7 +46,8 @@ def log(self, action, extra=None, save=False):
             'project': self.node.parent_id,
             'node': self.node._primary_key,
             'folder_id': self.node.get_addon('box', deleted=True).folder_id,
-            'folder': self.node.get_addon('box', deleted=True).folder_name
+            'folder_name': self.node.get_addon('box', deleted=True).folder_name,
```
Comment (sloria) [inline website/addons/box/utils.py]: To clarify: we can keep the `folder` key as is; just add `folder_path`.

**Comment by sloria** [comment]
Pass finished. Just a minor change.

File: website/addons/box/utils.py (hunk 2 above)
Comment (chennan47) [inline website/addons/box/utils.py]: fixed in 8446aca

**Pull request merged**